searchlib/src/vespa/searchlib/attribute/attrvector.hpp

```diff
@@ -89,6 +91,36 @@ NumericDirectAttrVector(const vespalib::string & baseFileName)
     }
 }
 
+template <typename F, typename B>
+long
+NumericDirectAttrVector<F, B>::onSerializeForAscendingSort(DocId doc, void* serTo, long available, const search::common::BlobConverter* bc) const
+{
+    if (!F::IsMultiValue()) {
+        return search::NumericDirectAttribute<B>::onSerializeForAscendingSort(doc, serTo, available, bc);
+    }
+    search::attribute::NumericSortBlobWriter<BaseType, true> writer;
```

Comment:
Consider adding a helper function so lines 101-106 can be shared between this function and _onSerializeForDescendingSort()_.

Reply:
Done.